libnmstate/nm/applier.py

```diff
@@ -246,6 +252,10 @@ def _get_new_ifaces(con_profiles):
         ifname = con_profile.devname
         nmdev = device.get_device_by_name(ifname)
         if not nmdev:
+            # When the profile id is different from the iface name, use the
```

Comment:
AFAIU, you could just always use `con_profile.con_id` instead. But then it is also strange that the method is named liked it will return interfaces but it can also return connection IDs.

Reply:
I mean to use this code:

```python
if not nmdev:
    ifaces_without_device.add(con_profile.con_id)
```

AFAIU, it has the identical result as the current code. To me the current code does not make this more clear.

Reply:
Line 252 can then be removed.

Reply:
I guess this function should be named `_get_profiles_for_new_interfaces()`.